File: 1672-richest-customer-wealth/1672-richest-customer-wealth.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int max = 0;
        for (int i=0; i<accounts.size(); i++) {
            int answer = 0;
            for (int j =0; j<accounts[i].size(); j++) {
                answer += accounts[i][j];
            }
            if (max < answer) max = answer;
        }
        return max;
    }
};

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.py
class Solution(object):
    def maximumWealth(self, accounts):
        max = 0
        for i in range(len(accounts)):
            result = sum(accounts[i])
            if (result > max):
                max = result
        return max

////////////////////////////////
// another efficient solution //
class Solution(object):
    def maximumWealth(self, accounts):
        return max(sum(acc) for acc in accounts)
